README.md

```diff
@@ -39,13 +39,13 @@ conda install -c conda-forge rclone
 
 Next [rclone must be linked to dropbox](https://rclone.org/dropbox/). The next steps will use the remote name specified during the linking process. For the following examples we will use the name `dropbox`. If you forgot the name, you can use `rclone listremotes`.
 
-Before syncing the workspace data, it is recommended to remove any empty job folders, as these will overwrite the data on dropbox. One way to do this is to remove the job folders of unused engines. We provide a bash script which can help. For example, to remove job folders for all engines EXCEPT hoomd, run:
+Before syncing the workspace data, it is recommended to remove any empty job folders, as these will overwrite the data on dropbox. One way to do this is to remove the job folders of unused engines. We provide a bash script which can help(located in `reproducibility_study/reproducibility_project/bin/clean_by_engine.sh`). For example, to remove job folders for all engines EXCEPT hoomd, run:
 
 ```
-bash clean_by_engine.sh hoomd
+bash bin/clean_by_engine.sh hoomd
 ```
 
-Be careful when using this script as it could delete your data! Before using it is advisable to make a backup and make sure that you spell/capitalize your engine the same way as in the statepoint.
+Be careful when using this script as it could delete your data! Make sure so check which project (`mosdef_reproducibility` or `lrc_shift_subproject`) you are in using `signac project`. Before using it is advisable to make a backup and make sure that you spell/capitalize your engine the same way as in the statepoint.
```

Comment:
```suggestion
Be careful when using this script as it could delete your data! Make sure to check which project (`mosdef_reproducibility` or `lrc_shift_subproject`) you are in using `signac project`. Before using it is advisable to make a backup and make sure that you spell/capitalize your engine the same way as in the statepoint.
```